### PR TITLE
Include `use_helpers` in all components

### PR DESCRIPTION
**Pull request by reeganviljoen**
### What are you trying to accomplish?

In order to complete #1976 the `#use_helpers` method needs to be added to components by default, so this pr is part of a two part series, hopefully this can bee included with version 4

### CODE REVIEW

File: test/sandbox/app/components/use_helpers_component.rb [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # frozen_string_literal: true
 
 class UseHelpersComponent < ViewComponent::Base
-  include ViewComponent::UseHelpers
-
```
Comment (boardfish) [inline test/sandbox/app/components/use_helpers_component.rb:5]:
```suggestion
  # include ViewComponent::UseHelpers
  # This module is now part of ViewComponent::Base, but the tests that
  # use this component are testing the functionality the module provides.
```

Comment (reeganviljoen) [reply test/sandbox/app/components/use_helpers_component.rb:5]: @boardfish I added this

**Review by joelhawksley — APPROVED** [review]

**Comment by reeganviljoen** [comment]
@joelhawksley I have applied you changes, thank you 👍

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ nav_order: 5
 
 ## main
 
+* Include ViewComponent::UseHelpers by default instead.
```
Comment (Spone) [inline docs/CHANGELOG.md:13]: Instead of what?

Comment (joelhawksley) [inline docs/CHANGELOG.md:13]:
```suggestion
* Include ViewComponent::UseHelpers by default.
```

Comment (reeganviljoen) [reply docs/CHANGELOG.md]: thank you @Spone, @joelhawksley has fixed this

**Comment by reeganviljoen** [comment]
@joelhawksley, @Spone , @boardfish any updates

**Pull request merged**